### PR TITLE
Doc/synchronize

### PR DESCRIPTION
**Pull request by KarimGl**
synchronize scenarios and environments

### CODE REVIEW

File: docs/getting_started/write.md [hunk 1]
```diff
@@ -57,6 +57,32 @@ Then, create a file `environment.json` with the following content :
...
+!!! tip "An environment is already defined on your Chutney server?"
+    If you have already defined an environment on your Chutney server, you can import it by calling `synchroniseLocal` function of `EnvironmentSynchronizeService`:
+    
+    ``` kotlin
+        import EnvironmentSynchronizeService
+        import com.chutneytesting.kotlin.util.ChutneyServerInfo
+        
+        fun main() {
+            val chutneyLocalServer = ChutneyServerInfo(
+                remoteServerUrl = "http://localhost:8081",
```
Comment (nbrouand) [inline docs/getting_started/write.md:69]: https ?

File: docs/installation/ci_cd/scenario_sync.md [hunk 2]
```diff
@@ -35,14 +17,60 @@ val chutneyLocalServer = ChutneyServerInfo(
...
+        }
+        Then("I am on the front page") {
+            SuccessAction()
+        }
+    }
+    ```
+
+    Now your scenario is created on the server ! :material-rocket:
+
+## update your scenario on chutney server
```
Comment (nbrouand) [inline docs/installation/ci_cd/scenario_sync.md:66]: Update

File: docs/installation/ci_cd/scenario_sync.md [hunk 3]
```diff
@@ -52,6 +80,5 @@ Now, the last step before running your scenario from the server is to setup envi
 
     To manage environments on a server, you can :
 
-    * Use the UI forms,
-    * Copy an existing environment file into the server configuration folder (default is `.chutney/conf/environment`),
-    * Import an existing environment file, (// TODO check bug on model compatibilty Kotlin/Server)
+    * Use the UI forms to create or import environments.
+    * Copy an existing environment file into the server configuration folder (default is `.chutney/conf/environment`)
```
Comment (nbrouand) [inline docs/installation/ci_cd/scenario_sync.md:84]: May be add a tooltip to explain the synchronize can also be done in a CI/CD (and i think it should be the best way to do)

File: docs/getting_started/write.md [hunk 4]
```diff
@@ -57,6 +57,32 @@ Then, create a file `environment.json` with the following content :
     ]
 }
 ```
+!!! tip "An environment is already defined on your Chutney server?"
+    If you have already defined an environment on your Chutney server, you can import it by calling `synchroniseLocal` function of `EnvironmentSynchronizeService`:
```
Comment (bessonm) [inline docs/getting_started/write.md:61]: It has nothing to do with the doc, but we have a difference in spelling with synchroni**z**e and synchroni**s**e

File: docs/getting_started/write.md [hunk 5]
```diff
@@ -57,6 +57,32 @@ Then, create a file `environment.json` with the following content :
...
+            environmentSynchronizeService.synchroniseLocal(serverInfo = chutneyLocalServer)
+       }
+    ```
+    `synchroniseLocal` function parameters:
+
+    | name               | type                       | Required  | default                 |          description                                     |
+    |:-------------------|:---------------------------|:---------:|:------------------------|:---------------------------------------------------------|
+    | `serverInfo`       | `ChutneyServerInfo`        |     *     |                         | holds needed informations to connect to chutney server   |
+    | `environmentsPath` | `String`                   |           | `.chutney/environments` | where to save the imported environments                  |
+    | `force`            | `Boolean`                  |           | `false`                 | if true, locally existing environments files will be overridden.</br>Else they will be keeped unchanged |
```
Comment (bessonm) [inline docs/getting_started/write.md:85]: keeped -> kept

File: docs/getting_started/write.md [hunk 6]
```diff
@@ -57,6 +57,32 @@ Then, create a file `environment.json` with the following content :
...
+            val environmentSynchronizeService = EnvironmentSynchronizeService()
+    
+            environmentSynchronizeService.synchroniseLocal(serverInfo = chutneyLocalServer)
+       }
+    ```
+    `synchroniseLocal` function parameters:
+
+    | name               | type                       | Required  | default                 |          description                                     |
+    |:-------------------|:---------------------------|:---------:|:------------------------|:---------------------------------------------------------|
+    | `serverInfo`       | `ChutneyServerInfo`        |     *     |                         | holds needed informations to connect to chutney server   |
```
Comment (bessonm) [inline docs/getting_started/write.md:83]: information**s** -> information
it is like water, uncountable ;)

**Review by bessonm — DISMISSED** [review]

**Review by boddissattva — APPROVED** [review]

**Review by nbrouand — APPROVED** [review]

**Pull request merged**